From 29aabaaab2efc4f0378f0a3d7a31e4534283e584 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Wed, 24 Mar 2021 18:12:20 -0700
Subject: [PATCH] Drop system assimp dependency to force vendoring

There's a problem with the assimp 5.0.1 CMake targets that causes them
to fail on systems that use LIB_SUFFIX, like RedHat distributions do.
Unfortunately, the failure causes find_package(assimp) to fail even if
QUIET is specified.

Dropping the dependency (for now) will force the vendor package to build
from source instead.
---
 rpm/ros-rolling-rviz-assimp-vendor.spec | 2 --
 1 file changed, 2 deletions(-)

diff --git a/rpm/ros-rolling-rviz-assimp-vendor.spec b/rpm/ros-rolling-rviz-assimp-vendor.spec
index 2c97ea01..1a141761 100644
--- a/rpm/ros-rolling-rviz-assimp-vendor.spec
+++ b/rpm/ros-rolling-rviz-assimp-vendor.spec
@@ -14,9 +14,7 @@ License:        Apache License 2.0
 URL:            http://assimp.sourceforge.net/index.html
 Source0:        %{name}-%{version}.tar.gz
 
-Requires:       assimp-devel
 Requires:       ros-rolling-ros-workspace
-BuildRequires:  assimp-devel
 BuildRequires:  ros-rolling-ament-cmake
 BuildRequires:  ros-rolling-ament-cmake-lint-cmake
 BuildRequires:  ros-rolling-ament-cmake-xmllint
-- 
2.25.1

